tests/src/Functional/SmsFrameworkGatewayAdminTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\sms\Functional;
+
+use Drupal\Core\Url;
+use Drupal\sms\Entity\SmsGateway;
+
+/**
+ * Tests manage event access page.
```

Comment:
I disagree, the class is still dedicated to gateway admin tests. Its where the old gateway admin web tests should be migrated to.